mainnet-1/specs/stargaze.json

```diff
@@ -0,0 +1,780 @@
+{
+    "proposal": {
+        "title": "Add Specs: Stargaze",
+        "description": "Adding new specification support for relaying agoric data on stargaze",
+        "specs": [
+            {
+                "index": "STRGZ",
+                "name": "stargaze mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL"
```

Comment:
stargaze is using cosmossdk v50
the specs needs to import from cosmossdk50

Reply:
Fix it